scania-digital-design-system/tegel
# feat(readme): adding missing prefix to spinner and dropdown

**Describe pull-request**  
Missing prefix in two readmes. 

**Solving issue**  
Fixes: [DTS-1815](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1815)

**How to test**  
1. Go to Storybook
2. Check in Components -> Spinner and Dropdown 
3. See that the prefix is present under Notes tab
